src/server/json_family.cc

```diff
@@ -2233,7 +2233,8 @@ void JsonFamily::Get(CmdArgList args, ConnectionContext* cntx) {
 // TODO: Add sensible defaults/categories to json commands
 
 void JsonFamily::Register(CommandRegistry* registry) {
-  constexpr size_t kMsetFlags = CO::WRITE | CO::DENYOOM | CO::FAST | CO::INTERLEAVED_KEYS;
+  constexpr size_t kMsetFlags =
+      CO::WRITE | CO::DENYOOM | CO::FAST | CO::INTERLEAVED_KEYS | CO::NO_AUTOJOURNAL;
```

Comment:
Maybe I am missing something but from `transaction.cc`

```
  1363   // If autojournaling was disabled and not re-enabled the callback is writing to journal.
  1364   // We do not allow preemption in callbacks and therefor the call to RecordJournal from
  1365   // from callbacks does not allow await.
  1366   // To make sure we flush the changes to sync we call TriggerJournalWriteToSink here.
  1367   if ((cid_->opt_mask() & CO::NO_AUTOJOURNAL) && !re_enabled_auto_journal_) {
  1368     TriggerJournalWriteToSink();
  1369     return; 
  1370   }
```

So when `CO::NO_AUTOJOURNAL` is `added` then it means we also write to the `journal` the opposite of the claim that we write it twice no ?

Reply:
I didn't get what do you want to say, but the code that you have added is  journal->RecordEntry(0, journal::Op::NOOP, 0, 0, nullopt, {}, true);